code/game/area/Space Station 13 areas.dm

```diff
@@ -67,7 +68,7 @@ var/list/teleportlocs = list()
 
 /proc/process_teleport_locs()
 	for(var/area/AR in world)
-		if(istype(AR, /area/shuttle) || istype(AR, /area/wizard_station)) continue

```

Comment:
the istype/area/shuttle was there to prevent wizard going on syndicate shuttle, abandonned shuttle, supply shuttle at centcome, emergency shuttle at centcom.